sdv/multi_table/hma.py

```diff
@@ -349,6 +350,11 @@ def _augment_table(self, table, tables, table_name):
                     foreign_key,
                     progress_bar_desc
                 )
+                for column in extension.columns:
+                    self.extended_columns[child_name][column] = FloatFormatter(
+                        enforce_min_max_values=True)
+                    self.extended_columns[child_name][column].fit(extension, column)
```

Comment:
I think nan values might be an issue here - if we can't get parameters for a child they end up being nan values here, and then we clear the whole augmented table of nans in line 366. If we move fitting the float formatters to after that we should be good.

Reply:
@frances-h I added a line to handle the case where all values are `nan`. The FloatFormatter is only used to store the min/max values, so it should work as long as there's one non-nan number.

Reply:
Right but if there's NaNs in the input, we'll end up adding NaNs back on the reverse as well, no?